test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetAllThreads/allthr001/allthr001.cpp

```diff
@@ -149,12 +167,24 @@ jint  Agent_Initialize(JavaVM *jvm, char *options, void *reserved) {
     return JNI_OK;
 }
 
+void releaseThreadInfo(JNIEnv *env, jvmtiThreadInfo *info) {
+    jvmti->Deallocate((unsigned char *)info->name);
+    if (info->thread_group != NULL) {
+        env->DeleteLocalRef(info->thread_group);
+    }
+    if (info->context_class_loader != NULL) {
+        env->DeleteLocalRef(info->context_class_loader);
+    }
+}
+
 JNIEXPORT void checkInfo(JNIEnv *env, int ind) {
     jint threadsCount = -1;
     jthread *threads;
-    int i, j, found;
+    int i, j;
+    bool found;
```

Comment:
Nit: this isn't old style C code, we don't have to declare all variables up front. New variables should be declared where needed.

Reply:
ok :)
fixed.